js/scripts/zone_loaders/end_zone_loader.js

```diff
@@ -69,42 +134,46 @@ const loadEndZone = () => {
+        //     new DrillBot(Vector.blockToWorldSpace(portal1Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal1Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal1Coordinates)),
+        //     ]);
+        // }
 
-        const portal2Coordinates = new Vector(70, 23);
-        const portal2 = new Portal(portal2Coordinates, Portal.YELLOW);
-        GAME.addEntity(portal2);
-        if (STORY.botsKilled < 20) {
-            portal2.fillWithEnemies([new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)), 
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                    new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
-                                ]);
-        }
-        GAME.addEntity(new Wizard(Vector.blockToWorldSpace(new Vector(90, 32)), 
+        // const portal2Coordinates = new Vector(70, 23);
+        // const portal2 = new Portal(portal2Coordinates, Portal.YELLOW);
+        // GAME.addEntity(portal2);
+        // if (STORY.botsKilled < 20) {
+        //     portal2.fillWithEnemies([new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     new DrillBot(Vector.blockToWorldSpace(portal2Coordinates)),
+        //     ]);
+        // }
+        GAME.addEntity(new Wizard(Vector.blockToWorldSpace(new Vector(85, 32)),
             null));
-        GAME.addEntity(new MamaChad(Vector.blockToWorldSpace(new Vector(85, 29))));
+        GAME.addEntity(new MamaChad(Vector.blockToWorldSpace(new Vector(82, 25))));
+        ASSET_MGR.playMusic(MUSIC.END.path, MUSIC.END.volume);
+
+        setTimeout(() => {
+        }, 1000);
```

Comment:
I was thrown off by the setTimeout being used for music. If it's not necessary for functionality of playing music I prefer not using a setTimeout. I forgot to remove it it would appear.